lib/core/errors.h

```diff
@@ -92,6 +96,10 @@ void internal_logging(int log, const char * format, ...);
 // Should be used sparingly, and limited to useful areas.
 #define INFO(m, a...) if (__log_level > 2) { internal_logging(LOG_INFO, m, ## a); }
 
+// Use this for fatal errors that kotekan can't recover from.
+// Prints an error message and raises a SIGINT.
+#define FATAL_ERROR(message, a...) { ERROR(message); exit_kotekan(ReturnCode::FATAL_ERROR);}
```

Comment:
I don't think this works with variadic calls to `FATAL_ERROR`, i.e. `FATAL_ERROR("Error value %d", myval);`.  Not completely sure my suggestion works either since this is compound macro, but this should be tested regardless. 
```suggestion
#define FATAL_ERROR(message, a...) { ERROR(message, ## a); exit_kotekan(ReturnCode::FATAL_ERROR);}
```